src/app/features/collectibles/components/ordinals.tsx

```diff
@@ -26,15 +27,23 @@ function Inscription({ path }: InscriptionProps) {
 
   const inscription = whenOrdinalType(data['content type'], {
     image: () => ({
-      type: 'image',
-      title: data.title,
       infoUrl: createInfoUrl(data.content),
       src: `https://ordinals.com${data.content}`,
+      title: data.title,
+      type: 'image',
     }),
+    text: () => {
```

Comment:
Just to note that this helper doesnt have to be described as fn, could be the object directly